### PR TITLE
[WIP] New video layout

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

We want to test a new video page layout, that we think works particularly good with series, and try to time it with the launch of [Anywhere but Westminster](http://www.theguardian.com/commentisfree/series/anywhere-but-westminster).
## What is the value of this and can you measure success?

We will be measuring:
- Video 75% completes (to correspond with the larger player)
- Onward video journeys (to test the related container stuff)
## Does this affect other platforms - Amp, Apps, etc?

Nope
## Screenshots

Coming soon
## Request for comment

@akash1810 @blongden73

### CODE REVIEW

File: applications/app/views/fragments/mediaBodyImmersive.scala.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                                            </div>
+                                        </div>
+                                    }
+                                    case _ => { }
+                                }
+                        </div>
+                    </div>
+                </div>
+
+                @media match {
```
Comment (akash1810) [inline applications/app/views/fragments/mediaBodyImmersive.scala.html]: we don't seem to be doing anything here...

Comment (jamesgorrie) [inline applications/app/views/fragments/mediaBodyImmersive.scala.html]: 🚧

File: static/src/stylesheets/module/content/_media.global.scss [hunk 2]
```diff
@@ -35,8 +92,69 @@
...
+            .video__share {
+                float: left;
+                width: gs-span(1.75);
+            }
+            .button {
+                &--more {
+                    background-color: transparent;
+                    border-color: rgba(255, 255, 255, .5);
+                }
+            }
```
Comment (OliverJAsh) [inline static/src/stylesheets/module/content/_media.global.scss]: Lots of specificity being added here. Can we avoid doing that?

**Comment by jamesgorrie** [comment]
Closing as we are no longer doing it.

File: applications/app/views/media.scala.html [hunk 3]
```diff
@@ -1,7 +1,8 @@
 @(page: MediaPage)(implicit request: RequestHeader)
 
 @main(page){ }{
-
-    @fragments.mediaBody(page)
```
Comment (OliverJAsh) [inline applications/app/views/media.scala.html]: Is this fragment still needed?

**Comment by OliverJAsh** [comment]
Nice work!

I noticed there are 132 CSS additions. Were you not able to remove any CSS used by the old video layout?

**Pull request closed without merging**